### ISSUE TITLE
Civic Tech & Design Reader

### ISSUE THREAD

**Issue by mateoclarke** [open-austin/project-ideas#101]
### What problem are we trying to solve?

There are has been a lot written about Civic Tech, this would be a community curated collection of articles to give folks a common baseline and and understanding of what is happen globally in the open data and civic tech movements.

If done right, it could be a good landing place for new folks and could foster cross brigade collaborations.

### Who will benefit (directly and indirectly) from this project?

Readers. New civic technologists and designers.

### Where can we find any research/data available/articles?

Links here.

### What help is needed at this time?

Suggesting articles. Deciding the format.

### What are the next steps (validation, research, coding, design)?

Collect articles. Create a Github repo

Here is a start:
https://medium.com/@jbiroscak/https-medium-com-jbiroscak-how-civic-tech-volunteers-should-engage-government-staff-d3a9f8eed3bd
http://codeforamerica.github.io/fellowship-reader-2015/
http://codeforamerica.github.io/fellowship-reader-2016/
https://docs.google.com/document/d/1ogE8XV15T-caJ4UT-q5BlzUewtZsMF0DESl-cpLqqNk/pub

----

#### Checklist for NEW ideas :baby:
Hey, you're official! You're now part of the growing civic hacking community in Austin. Here's a few things to get started (a couple you've probably already done).

- [x] Create this idea issue
- [x] Flesh out the who, where, and what questions above
- [ ] Start the conversation about this new idea on [Slack](https://openaustin-slackin.herokuapp.com/)!
- [ ] Respond with at least one update on this issue within the next month

#### Checklist for ACTIVE projects :fire:
Let's get this project started! When this idea starts taking off, the Projects Core Team will start helping this project's lead(s) out with project management and connecting you to resources you may need. To get there, please complete and check off the following:

- [ ] Complete [Open Leadership 101](https://mozilla.teachable.com/p/open-leadership-101)
- [ ] Create a [GitHub repository](https://github.com/new) and Slack channel for work.
- [ ] Create a [README file](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/) in your project repository. This file should help newcomers understand what your project is, why it's important, and kinds of help you're looking for.
- [ ] [Create issues](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/project-set-up-for-collaboration-with-github/#assignment--add-your-first-issue) to describe each task that you plan to do or need help with and how a contributor can get started on that task. You might start and stop a lot, so consider issues as your to-do list.

#### Checklist for FEATURED Projects :tada:
To have your project FEATURED on [Open-Austin.org](https://open-austin.org/), complete the following documentation. In past projects, well-documented featured projects have more contributions than other projects.

* [ ] In your README, link to the [Open Austin Community Participation Guidelines](https://docs.google.com/document/d/1OujyBccPpepUXSY5_nP3alzvd81WlCKSeSf_-sU5K-U/edit?usp=sharing) or write your own code of conduct.
* [ ] [Create file: LICENSE](http://choosealicense.com/) to give your project an open license, allowing for sharing and remixing.
* [ ] [Create file: CONTRIBUTING.md](https://github.com/acabunoc/mozsprint-repo-template/blob/master/CONTRIBUTING.md) so others know [how they can contribute](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-contributor-guidelines/).

Once all of the above is complete,
- [ ] [Create an issue](https://github.com/open-austin/open-austin.github.io/issues/new) on the open-austin.github.io repo with the title `Add [this project] to projects page. An Open Austin leader will review this issue and post your project :balloon:

If you get stuck at any point, feel free to reach out to the leadership team on Slack by adding @leadership to your message. We're here to help you make real changes to our city.

**Comment by mateoclarke** [comment]
Just learned about this motherload: https://docs.google.com/spreadsheets/d/1FzmvVAKOOFdixCs7oz88cz9g1fFPHDlg0AHgHCwhf4A/edit#gid=963594345

cc/ @amenity

**Comment by werdnanoslen** [comment]
There's also the Civic Quarterly https://civicquarterly.com/

**Comment by werdnanoslen** [comment]
There's another project in Houston about getting RSS feeds together https://github.com/sketch-city/project-ideas/issues/46

Perhaps an MVP here could be something like our [#civic-tech-news](https://open-austin.slack.com/messages/C72U14JA1) Slack channel

**Comment by werdnanoslen** [comment]
I think this can also be useful for #34
